tests/e2e/specs/dashboard/notifications.test.js

```diff
@@ -18,56 +18,92 @@ const goToSiteKitDashboard = async () => {
 };
 
 describe( 'core site notifications', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'e2e-tests-auth-plugin' );
-		await setSiteVerification();
-		await setSearchConsoleProperty();
-	} );
-
-	it( 'displays core site notifications on the main dashboard', async () => {
-		// Add the test notification (by default there are none).
-		await wpApiFetch( {
-			path: 'google-site-kit/v1/e2e/core/site/notifications',
-			method: 'post',
-			data: testSiteNotification,
+	// The proxy test cannot currently be done and needs to be skipped TODO tests need to to be fixed to handle proxy tests.
+	describe.skip( 'when using proxy', () => {
```

Comment:
Can you clarify why we cannot currently test this? I'd think we can mock the API response? We should be able to simulate the proxy being used by filtering in proxy credentials. Thoughts @aaemnnosttv?

Reply:
We don't currently have the infrastructure in place to fake proxy authentication in E2E. We could add it here, but I thought it would be a bit out-of-scope for this issue and is actually a symptom of a larger problem. I created an issue for it here: https://github.com/google/site-kit-wp/issues/1503